otodom.py

```diff
@@ -0,0 +1,362 @@
+# python modules
+import json
+import os
+import re
+import unicodedata
+from urllib.parse import urlparse, quote
+
+
+# third party
+import requests
+from bs4 import BeautifulSoup
+from scrapper_helpers.utils import caching
+
+# own modules
+
+BASE_URL = 'http://www.otodom.pl'
+SCRAPE_LIMIT = os.environ.get('SCRAPE_LIMIT', None)
+
+WHITELISTED_DOMAINS = [
+    'otodom.pl',
+    'www.otodom.pl',
+]
+
+
+def replace_all(text, dic):
+    for i, j in dic.items():
+        text = text.replace(i, j)
+    return text
+
+
+def get_region_from_autosuggest(region_part):
+    url = "https://www.otodom.pl/ajax/geo6/autosuggest/?data={0}".format(region_part)
+    response = json.loads(get_response_for_url(url).text)[0]
+    type = response["level"]
+    text = response["text"].replace("<strong>", "").replace("</strong>", "").split(", ")
```

Comment:
The position of city in this list depends on the "level". If it's "CITY", city is in text[0], if it's "DISTRICT", city is in text[1]. Using variables here would only add to the confusion